covid19-etl/etl/idph_regional_icu_capacity.py

```diff
@@ -0,0 +1,136 @@
+
+        self.program_name = "open"
+        self.project_code = "IDPH-Regional_ICU_Capacity"
+        self.metadata_helper = MetadataHelper(
+            base_url=self.base_url,
+            program_name=self.program_name,
+            project_code=self.project_code,
+            access_token=access_token,
+        )
+
+        self.country = "US"
+        self.state = "IL"
+        self.etlJobDate = datetime.date.today().strftime("%Y-%m-%d")
+        self.summary_locations = []
+        self.summary_clinicals = []
+
+    def files_to_submissions(self):
+        """
+        Reads JSON file and convert the data to Sheepdog records
+        """
+
+        today = datetime.date.today()
+        latest_submitted_date = self.metadata_helper.get_latest_submitted_date_idph()
+        if latest_submitted_date == today:
+            print("Nothing to submit: today and latest submitted date are the same.")
+            return
+        today_str = today.strftime("%Y%m%d")
+
+        print(f"Getting data for date: {today_str}")
+        url = "https://idph.illinois.gov/DPHPublicInformation/api/COVIDExport/GetHospitalizationResultsRegion"
+        self.parse_file(latest_submitted_date, url)
+
+    def parse_file(self, latest_submitted_date, url):
+        """
+        Converts a JSON files to data we can submit via Sheepdog. Stores the
+        records to submit in `self.summary_locations` and `self.summary_clinicals`.
+
+        Args:
+            latest_submitted_date (date): the date of latest available "summary_clinical" for project
+            url (str): URL at which the JSON file is available
+        """
+        print("Getting data from {}".format(url))
+        with closing(self.get(url, stream=True)) as r:
+            data = r.json()
+            date = self.etlJobDate
+
+            for region in data:
+                (summary_location, summary_clinical) = self.parse_region(date, region)
+                self.summary_locations.append(summary_location)
+                self.summary_clinicals.append(summary_clinical)
+
+    def parse_region(self, date, hospital_region):
+        """
+        From county-level data, generate the data we can submit via Sheepdog
+        """
+        region = hospital_region["region"]
+        region_description = hospital_region["region_description"]
+
+        summary_location_submitter_id = format_submitter_id(
+            "summary_location",
+            {
+                "project": "idph_regional_icu_capacity",
+                "country": self.country,
+                "state": self.state,
+                "region": region,
+            },
+        )
+
+        summary_location = {
+            "country_region": self.country,
+            "submitter_id": summary_location_submitter_id,
+            "projects": [{"code": self.project_code}],
+            "province_state": self.state,
+            "state_hospital_region": region,
+            "state_region_description": strip_prefix(region_description),
```

Comment:
i don't see notes in the value mapping spreadsheet about this, could you provide an example of value for which you need to strip the prefix?

Reply:
This was used in the IDPH-Hospital ETL ([here](https://github.com/uc-cdis/covid19-tools/blob/1ac67e3d0b3639be585e21b68dee0137fdbde37f/covid19-etl/etl/idph_hospital.py#L17-L20)) . I assumed there could be a scenario where it could be used. 

Not something I've created! I can take it down if it is for the better!

Reply:
oh okay 🤔 could you check if the `strip_prefix` function output is ever different from the input? since we changed URLs, the data might have been cleaned up on their side

Reply:
For now, I see no difference. I wonder why was it placed in the first place. For now all we get is of the format -- "Region #" and the output is exactly the same. Should I go ahead and take it off. We can always update it if we think there is anything that we're missing.

Reply:
mm you know what, let's keep it, i was just curious